### DISCUSSION TITLE
My Meshroom is broke i guess

### DISCUSSION THREAD

**Discussion by XelerB** [alicevision/Meshroom#2330] [Q&A]
I think my meshroom is broke because i can't explain why it doesen't render the model, i do the model of my shoe but when i clik on  start it works but stop in meshing, the directory and the molders don't include spaces or other type of signs that don't can be in the directory, who can  help me?
![Desktop Screenshot 2024 03 01 - 16 42 56 07](https://github.com/alicevision/Meshroom/assets/161859886/fc6d2249-b8f4-45b2-8053-52256f102dc1)

**Comment by natowi** [comment]
Only 2 out of 53 images matched. Ideally all images should match. Get far closer to the shoe and make sure it is well lit. Make yourself familiar with the best practices on how to capture images for photogrammetry.
